examples/describe_consumer_groups.c

```diff
@@ -164,15 +164,29 @@ print_group_member_info(const rd_kafka_MemberDescription_t *member) {
             rd_kafka_MemberDescription_host(member));
         const rd_kafka_MemberAssignment_t *assignment =
             rd_kafka_MemberDescription_assignment(member);
-        const rd_kafka_topic_partition_list_t *topic_partitions =
+        const rd_kafka_topic_partition_list_t *assigned_topic_partitions =
             rd_kafka_MemberAssignment_partitions(assignment);
-        if (!topic_partitions) {
+        const rd_kafka_MemberAssignment_t *target_assignment =
+            rd_kafka_MemberDescription_target_assignment(member);
+        const rd_kafka_topic_partition_list_t *target_topic_partitions =
+            rd_kafka_MemberAssignment_partitions(target_assignment);
```

Comment:
This function shouldn't check the argument is NULL, instead it mustn't be called with a NULL argument.
```suggestion
            target_assignment
                ? rd_kafka_MemberAssignment_partitions(target_assignment)
                : NULL;
```